tests/test_epics_attenuator.py

```diff
@@ -1,60 +1,63 @@
-############
-# Standard #
-############
 import logging
 
-###############
-# Third Party #
-###############
-import pytest
 from unittest.mock import Mock
+from ophyd.status import wait as status_wait
 
-##########
-# Module #
-##########
-from pcdsdevices.sim.pv import  using_fake_epics_pv
+from pcdsdevices.sim.pv import using_fake_epics_pv
 from pcdsdevices.epics import FeeAtt
 
+from .conftest import attr_wait_true
+
 logger = logging.getLogger(__name__)
 
-@using_fake_epics_pv
-@pytest.fixture(scope='function')
-def attenuator():
+
+def fake_att():
+    """
+    using_fake_epics_pv does cleanup routines after the fixture and before the
+    test, so we can't make this a fixture without destabilizing our tests.
```

Comment:
What about the cleanup destabilizes? The only reason we need the wrapper around the test is that some PVs are instantiated lazily. We could keep using fixtures and instead just make sure that the all the PVs are instantiated as `FakeEpicsPV`s by calling `wait_for_connection`?

Reply:
It literally breaks the class with the new implementation of FakeEpicsPV with no way of recovery. The end of the `using_fake_epics_pv` wrapper does such a thorough cleanup that the object will never work again.